proposals/record-structs.md

```diff
@@ -143,6 +143,8 @@ The record struct includes a synthesized method equivalent to a method declared
 public override string ToString();
 ```
 
+If the record's printable members do not include a readable property with a non-`readonly` `get` accessor, then the synthesized `PrintMembers` and `ToString` methods are `readonly`.
```

Comment:
The situation where a printable member is a non-`readonly` field should also be accounted for.

Reply:
```suggestion
If the record's printable members do not include a readable property with a non-`readonly` `get` accessor, then the synthesized `PrintMembers` and `ToString` methods are `readonly`. There is no requirement for the record's fields to be `readonly` for the `PrintMembers` and `ToString` methods to be `readonly`.
```